src/app/home/sponsors/sponsors.component.ts

```diff
@@ -1,15 +1,30 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+
+import { Observable, Subscription } from 'rxjs';
+
+import { Sponsor } from 'src/app/shared/models/sponsor.model';
+import { SponsorsService } from './sponsors.service';
+import { environment } from '../../../environments/environment';
 
 @Component({
   selector: 'gp-sponsors',
   templateUrl: './sponsors.component.html',
   styleUrls: ['./sponsors.component.scss']
 })
-export class SponsorsComponent implements OnInit {
+export class SponsorsComponent implements OnInit, OnDestroy {
+  subscription: Subscription;
+  sponsors$: Observable<Sponsor[]>;
+  baseUrl = environment.firebaseStorageUrl;
 
-  constructor() { }
+  constructor(private sponsorsService: SponsorsService) {}
 
   ngOnInit() {
+    this.sponsors$ = this.sponsorsService.getAll();
   }
 
+  ngOnDestroy(): void {
+    if (this.subscription) {
+      this.subscription.unsubscribe();
+    }
+  }
```

Comment:
This is not needed, you are not subscribing to anything, so you don't need to unsubscribe.

Reply:
I just realized this is also in activity-indicators components, could you get rid of it there too, please